src/rendering/typst.jl

```diff
@@ -39,7 +40,7 @@ function compile_typst(document::AbstractString)
             try
                 # `pipeline` is not yet supported for `TypstCommand`
                 redirect_stdio(stdout=out, stderr=err) do
-                    run(ignorestatus(typst`compile temp.typ`))
+                    run(ignorestatus(addenv(typst`compile temp.typ`, "TYPST_FONT_PATHS" => julia_mono)))
```

Comment:
I couldn't see where `julia_mono` is defined, might have missed it though.

Reply:
It's exported by Typstry, so it's loaded [here](https://github.com/JuliaPlots/MakieTeX.jl/blob/632c0d73886790ab4349b153ec81b1cb5907bd6c/src/MakieTeX.jl#L6). It could be made more explicit with `using Typstry: Typstry, TypstString, @typst_cmd, @typst_str, julia_mono`.

Reply:
```suggestion
                    run(ignorestatus(addenv(typst`compile temp.typ`, "TYPST_FONT_PATHS" => Typstry.julia_mono)))
```
should make it clearer IMO